### ISSUE TITLE
Custom maps not working and screen doing weird things in map menu

### ISSUE THREAD

**Issue by ZakChrom** [VerdantGameOfficial/issues#14]
**Describe the bug**
Custom maps not working

**To Reproduce**
Download a map
Put it in maps/
Go to game and press the maps button
The screen is forever fading and brightening
Click on map
No blocks are put to game scene

**Expected behavior**
Map working

**Screenshots**
null

**Desktop (please complete the following information):**
 - OS: Windows 8,1
 - Browser: no
 - Version: a-0.3.1

**Additional context**
json file:
{
	"meta": {
		"name": "Test map 1",
		"description": "Test map 1",
		"author": "Verdant"
	},
	"blockdata": [
		{"x": 0,"y": 0,"t": 0},
		{"x": 0,"y": 1,"t": 1},
		{"x": 1,"y": 1,"t": 1},
		{"x": 2,"y": 1,"t": 1},
		{"x": 2,"y": 0,"t": 2}
	]
}

**Logs**
ERROR: Error parsing JSON at line 0: Expected 'EOF'
   at: parse (core/bind/core_bind.cpp:3216) - Error parsing JSON at line 0: Expected 'EOF'

**Comment by ZakChrom** [comment]
Heres a video:
https://user-images.githubusercontent.com/88797993/167899550-61e12bc3-7b00-4842-988b-88b403711178.mp4

**Comment by Just-a-Unity-Dev** [comment]
Are you using NPP or VSCode to edit the file? If so please use CRLF line endings

**Comment by ZakChrom** [comment]
NPP

**Comment by ZakChrom** [comment]
but i downloaded the file and put it straight into maps without even opening it and it still didnt work

**Comment by Just-a-Unity-Dev** [comment]
[How about this tutorial?](https://m.youtube.com/watch?v=xkBERiYyplw)

**Comment by ZakChrom** [comment]
doesnt work
it didnt allow me to convert to crlf 
so i converted to lf and then to crlf but no changes

**Comment by Just-a-Unity-Dev** [comment]
How about LF?

**Comment by ZakChrom** [comment]
doesnt work
ill try cr too

**Comment by ZakChrom** [comment]
cr doesnt work either

**Comment by Just-a-Unity-Dev** [comment]
I honestly don't know anymore..

Try this:
```
{"meta":{"name":"Test map 1","description":"Test map 1","author":"Verdant"},"blockdata":[{"x":0,"y":0,"t":0},{"x":0,"y":1,"t":1},{"x":1,"y":1,"t":1},{"x":2,"y":1,"t":1},{"x":2,"y":0,"t":2}]}
```

**Comment by Just-a-Unity-Dev** [comment]
Actually, can you send the me the entire file for Test map 1?

**Comment by ZakChrom** [comment]
ill dm you it

**Comment by ZakChrom** [comment]
also the code doesnt work

**Comment by Just-a-Unity-Dev** [comment]
I'll work on this in a few hours, I need to sleep

**Comment by ZakChrom** [comment]
Good night

**Comment by Just-a-Unity-Dev** [comment]
Are you using the latest version? Even if you are, re-download it

**Comment by ZakChrom** [comment]
ill try

**Comment by ZakChrom** [comment]
nope doesnt work

**Comment by Just-a-Unity-Dev** [comment]
fixed in 0.4
